setup.py

```diff
@@ -21,7 +21,7 @@ def read(filename):
 
 setup(
     name='Henson-AMQP',
-    version='0.7.0',
+    version='0.8.0',
     author='Andy Dirnberger, Jon Banafato, and others',
```

Comment:
You should update the `author` field (and `AUTHORS` file) similar to [this](https://github.com/iheartradio/Henson/commit/65e14d3a2baeb506c992d3caf40ca3ddc0eddca8) PR.